applications/nrf_desktop/configuration/common/gpio_pins.h

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+#ifndef _GPIO_PINS_H_
+#define _GPIO_PINS_H_
+
+
+static const char * const port_map[] = {
+		DT_GPIO_P0_DEV_NAME,
+		DT_GPIO_P1_DEV_NAME
```

Comment:
we would need ifdef here to handle SoCs with 1 port only (e.g. 832)

Reply:
done